lib/api/wpc_proto.h

```diff
@@ -13,9 +13,33 @@
 /*
  * Maximum size of a response. It can be used as an hint
  * to give a big enough buffer to @ref WPC_Proto_handle_request
- * or @WPC_Proto_get_current_event_status
+ * note : it should be higher than this response size list, as it's a wp_GenericMessage
+ * wp_GetConfigsResp_size
+ * wp_GetGwInfoResp_size
+ * wp_SetConfigResp_size
+ * wp_SendPacketResp_size
+ * wp_GetScratchpadStatusResp_size
+ * wp_ProcessScratchpadResp_size
+ * wp_SetScratchpadTargetAndActionResp_size
+ * wp_UploadScratchpadResp_size
  */
-#define WPC_PROTO_MAX_RESPONSE_SIZE 512
+#define WPC_PROTO_MAX_RESPONSE_SIZE 450 
+
+/*
+ * Maximum size of an Event Status. It can be used as an hint
+ * to give a big enough buffer to @WPC_Proto_get_current_event_status
+ * or @onStackStatusReceived
+ * note : it should be higher than wp_StatusEvent_size, as it's a wp_GenericMessage
+ */
+#define WPC_PROTO_MAX_EVENTSTATUS_SIZE 550
```

Comment:
Maybe we could introduce WPC_PROTO_GENERIC_MESSAGE_OVERHEAD (to 10 or 20)
ANd this define would become: 
#define WPC_PROTO_MAX_EVENTSTATUS_SIZE   (wp_StatusEvent_size + WPC_PROTO_GENERIC_MESSAGE_OVERHEAD)

Reply:
So we have to add #include "config_message.pb.h" in this file, ok ?

Reply:
Then lest keep a hardcoded value